pkgs/top-level/all-packages.nix

```diff
@@ -5927,6 +5927,8 @@ with pkgs;
 
   gnuapl = callPackage ../development/interpreters/gnu-apl { };
 
+  gnu_shepherd = callPackage ../misc/gnu_shepherd { };
```

Comment:
```suggestion
  gnu-shepherd = callPackage ../misc/gnu-shepherd { };
```

should match pname

Reply:
Then I would favor gnu_sheperd. Are you fine with that?

Reply:
I recommend `gnu-shepard`.

```
nix-repl> gnu
gnu-cobol            gnucap               gnugrep              gnumeric             gnupg1compat         gnuradio             gnuradioMinimal      gnutls
gnu-config           gnucash              gnujump              gnunet               gnupg1orig           gnuradio3_7          gnuradioPackages     gnutls-kdh
gnu-efi              gnuchess             gnuk                 gnunet-gtk           gnupg22              gnuradio3_7Minimal   gnused
gnu-pw-mgr           gnuclad              gnulib               gnupatch             gnuplot              gnuradio3_7Packages  gnused_422
gnu-smalltalk        gnudatalanguage      gnum4                gnupg                gnuplot_aquaterm     gnuradio3_8          gnushogi
gnuapl               gnufdisk             gnumake              gnupg-pkcs11-scd     gnuplot_qt           gnuradio3_8Minimal   gnustep
gnubg                gnugo                gnumake42            gnupg1               gnupth               gnuradio3_8Packages  gnutar
```